csrc/runtime/executor.cpp

```diff
@@ -651,162 +692,212 @@ void KernelExecutor::initializeExecutorEntry(
       "Expected blockDim.x >= 32 but found ",
       launch_params.bdimx());
 
+  std::vector<GlobalBufferInfo> input_info;
```

Comment:
Construct input buffer info's as they're used to construct the binary args for the kernel.